templates/web/peterborough/waste/_more_services_sidebar.html

```diff
@@ -3,7 +3,7 @@ <h3>More services</h3>
   [% IF any_report_allowed AND NOT on_day_pre_5pm %]
     <li><a href="[% c.uri_for_action('waste/report', [ property.id ]) %]">Report a missed collection</a></li>
   [% END %]
-  [% IF any_request_allowed %]
+  [% IF ( NOT waste_features.request_disabled ) AND c.user_exists AND (c.user.is_superuser OR (c.user.from_body AND c.user.from_body.name == "Peterborough City Council")) %]
```

Comment:
Wonder if there's a better way to make these things staff-only, now we have a few scattered in the templates. Or at least some greppable comment each time.
Would make it easier to remove staff restriction when the time comes.

Reply:
This is only the 2nd I think which I copied from the first ;) Take your point about easier to remove, but didn't seem worth it if it was just going to go soon.

Reply:
Yeah that's fair enough!